website/docs/r/aviatrix_transit_gateway.html.markdown

```diff
@@ -137,6 +137,7 @@ In addition to all arguments above, the following attributes are exported:
 * `security_group_id` - Security group used for the transit gateway.
 * `cloud_instance_id` - Cloud instance ID of the transit gateway.
 * `private_ip` - Private IP address of the transit gateway created.
+* `ha_cloud_instance_id` - Cloud instance ID of the ha transit gateway.
```

Comment:
same

Reply:
All other ha based attributes are not in capital, like ha_subnet, ha_gw_size, and so on...